cpp/include/cudf/strings/strip.hpp

```diff
@@ -58,12 +58,12 @@ enum class strip_type {
  *
  * @throw cudf::logic_error if `to_strip` is invalid.
  *
- * @param strings Strings column for this operation.
+ * @param strings Strings column for this operation
  * @param stype Indicates characters are to be stripped from the beginning, end, or both of each
  * string. Default is both.
- * @param to_strip UTF-8 encoded characters to strip from each string.
+ * @param to_strip UTF-8 encoded characters to strip from each string
```

Comment:
```suggestion
 * @param to_strip UTF-8 encoded characters to strip from each string.
```
I don't suppose changes like this can be undone. This is getting frustrating to review.

Reply:
reverted after resolving conflicts. existing code change might have dots.